jekyll/add-ons.markdown

```diff
@@ -126,6 +132,45 @@ This approach enables all add-on responses to be captured in a single round of A
 
 ### Enhancing features
 
+There are two ways to enhance Ruby LSP features. One is handling DSLs that occur at a call site and that do not change
+which declarations exist in the project. A great example of this the Rails validation method, which accepts a symbol
+that represents a method that gets dynamically invoked. That style of DSL is what we refer to as a [call site
+DSL](#dealing-with-call-site-dsls).
```

Comment:
Would it be more correct to call this a 'DSL method' or something like that?

Reply:
I'm not sure what the best term is. A method that mutates the index like `belongs_to` is also a "DSL method".

I was looking for terms that can differentiate DSL on the call site vs DSLs that create declarations.

Reply:
I was meaning just the 'DSL' part, to me the 'DSL' is the _collection_ of all the available 'constructs'. Not a big deal though.